Add `Bespoke Music in osu!` news post (#11120)
ppy/osu-wiki: news/2024/2024-03-19-bespoke-music-in-osu.md

How did their story start? They were asked similar questions as Kasumi, so let's have two of the organisers take it away:

[![](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/Its_not_Marvin.png)](https://osu.ppy.sh/users/13600612)

> Hi, my name is Marvin, known as Its not Marvin in osu. I am mainly known to be the host of osu! Malaysia Tournament from 2020 to 2023.
> 
> It all started in 2021 with the previous host of osu! malaysia tournament, [Kumaxione](https://osu.ppy.sh/users/2582531), he wanted to do something special for his last year staffing, so he decided on commissioning a custom song with a local composer, [takehirotei](https://osu.ppy.sh/users/11793794).
> 
> What was challenging in particular, and definitely in 2023, was that the majority of us volunteering in local tourneys in Malaysia are uni students, so when physical classes restarted, the time and availability that felt like an abundance during the pandemic era is suddenly gone. To be able to plan and run the tourney with the limited time that we had, felt like quite the achievement, as it was exhausting and scary at times, wondering if we can finish things before the deadline of not.
> 

> > I feel like the taiko side definitely has larger budgets, and may have more sophisticated processes when it comes to the custom songs planning and producing process, as they had a custom song for every round of the [osu!taiko Malaysian Tournament 2022](/wiki/Tournaments/OTMT/2022), unlike std where we have 1-3 custom songs for the entire tournament

Reviewer: ```suggestion
> I feel like the taiko side definitely has larger budgets, and may have more sophisticated processes when it comes to the custom songs planning and producing process, as they had a custom song for every round of the [osu!taiko Malaysian Tournament 2022](/wiki/Tournaments/OTMT/2022), unlike std where we have 1-3 custom songs for the entire tournament.
```

> 
> I truly believe that this, along with the introduction of our mascot character, **IRIS**, helped a lot in weaving a recognizable visual identity for the whole tournament, which significantly enhanced the tournament experience.
> 
> ![osu!taiko Malaysia mascot "IRIS"](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/iris.jpg)

Visual arts go hand in hand with music commissions and tournament production, which is itself an interesting topic (that we may cover in the near future!).

> The music production process for our team is rather straightforward. We'll usually only specify the particular genre and difficulty that we're looking for to an artist based on the style that they're comfortable with, along with some example tracks provided. Besides that, the artists are given creative liberties to elevate it however they wish.
> 
> I think this process works very well as we also have a general channel for all our composers to mingle and share ideas for each other's works, allowing for more constructive opinions to be provided throughout the process by equally skilled individuals.

---

> The Malaysian tournaments collectively have 19 original tracks in total at the moment, but word is on the street that more is on the way!

Reviewer: when reading this for the first time i felt like i fumbled my words, maybe this reads better?
```suggestion
The Malaysian tournaments collectively have 19 original tracks in total at the moment, but the word on the street is that more is on the way!
```

## Inside the music pipeline

While [mangomizer](https://osu.ppy.sh/users/1893718) and [pishifat](https://osu.ppy.sh/users/3178418) touched on the process of World Cup music production, how does it fare for other tournaments? **Kasumi** had this to say:

> **Kasumi-sama:** The production process in GTS is very largely hands-off from us as admins. It all depends on the slot we shoot for, but for tiebreakers for example, we usually just give a general rundown of the structure we'd like, as well as a specific atmosphere we're shooting for. At that point, everything like genre, BPM and length, are basically decided by the artist's wish.
> 
> We only ask for relatively frequent drafts, in intervals of 30 seconds to 1 minute, so we can check if we first like the general vibe, as one would expect, and if we can map it accordingly to the planned difficulty, of course.

> The other tournament hosts interviewed here said basically the same thing, which means there's only one thing to do. (To ask some artists.)

Reviewer: i feel like having `(To ask some artists.)` in brackets is like explaining a joke right after you said it, i don't think its necessary?

Author: maybe, although to me it doesn't really sound like a joke

(applied)

[![](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/takehirotei.png)](https://osu.ppy.sh/users/11793794)

> Hey there, I'm [takehirotei](https://osu.ppy.sh/users/11793794), just your normal rhythm gamer that composes music as a side job. If you're from osu!, you might have heard of me from some tiebreaker songs from some tourneys such as osu! (and taiko) Malaysia Tournaments, [EGTS](/wiki/Tournaments/GTS#expert-global-taiko-showdown), or [4DMWC 2023](/wiki/Tournaments/4DM/2023) and many more, or if you're outside osu!, you might even heard me as the anime song remix guy for remixing some popular titles such as *Idol* (under the Imaginary Reality alias) or *Guitar*, *Loneliness* and *Blue Planet*.

[**takehirotei**](https://osu.ppy.sh/users/11793794) has produced over 20 songs for various community tournaments, 6 of them being for Global Taiko Showdown. Here's their perspective on how it all goes down:

> > For the production process, the host will set a deadline on the song, as well as some restrictions on the song (e.g. BPM limit, intensity, genres and more) in order to make sure the song will fit nicely to the planned (or not planned) slots. From time to time, we sometimes also send wips of the current state of the song to make sure that the song is going to the correct direction. This will ensures that we dont have to do unnessary changes later on when the full song is done, which can saves us a lot of time.

Reviewer: ```suggestion
> For the production process, the host will set a deadline on the song, as well as some restrictions on the song (e.g. BPM limit, intensity, genres and more) in order to make sure the song will fit nicely to the planned (or not planned) slots. From time to time, we sometimes also send wips of the current state of the song to make sure that the song is going to the correct direction. This will ensure that we dont have to do unnecessary changes later on when the full song is done, which can saves us a lot of time.
```

> I'd say not too different. The only difference I'd say is how the host handles the procedure. \[...] What's different is sometimes, the host might add you into the mapping channel, so that when the mapper has any questions for the composer, the composer can help them with it (e.g. "what snappings are in this part?", "what is the time signature for this part?" and so on).
> 
> One of the challenge that I've faced in making songs for osu! tournaments is to do a bit of research on the style the host is going to put me into. As there are many. For example, let's say if the host wants the song to be in DT1 slot for osu!taiko, then I need to do a bit of research on the songs that are usually used in that slot.
> 
> But if I'm lucky, sometimes the host will list down a long list of descriptions on how the song will go (thanks Kasumi), this will really help us composers to work on the correct direction. Another challenge that I'm facing while making songs for tournaments (or rhythm games in general) is that these kinds of songs usually don't have many chill parts, and sometimes it's a little bit challenging to make a song when most of the parts are intense parts.

> Regarding organisers giving detailed directions, mangomizer [explained in the video mentioned in the beginning](https://www.youtube.com/watch?v=0B6TyWML0zg&t=906s) that this is something the World Cups did initally, but moved away from due to it resulting in less cohesive songs.

Reviewer: ```suggestion
Regarding organisers giving detailed directions, mangomizer [explained in the video mentioned in the beginning](https://www.youtube.com/watch?v=0B6TyWML0zg&t=906s) that this is something the World Cups did initially, but moved away from due to it resulting in less cohesive songs.
```

> 
> My first tournament was actually [DDS Tournament](https://osu.ppy.sh/community/forums/topics/1418287?n=1) for which I offered a piano custom song, and then I decided to participate in [osu!taiko Malaysia Tournament](/wiki/Tournaments/OTMT) with my first tiebreaker custom song alongside with takehirotei's custom remix of my song being presented in [osu!mania LN Tournament 3](/wiki/Tournaments/o!mLN/3) ([Normal Distribution Remix](https://osu.ppy.sh/beatmapsets/1880819)), and I decided to develop the lore to go along with it and another custom song (for those who are interested, see [LNCryption](https://osu.ppy.sh/beatmapsets/1893946)) which was the reason I continued to do the custom songs in the tournament scenes.

Whether by having such a rooted connection between their musical identity and osu! or otherwise, HowToPlayLN does not lack insight into how the osu! tournament music scene clicks. These are their thoughts and advice on what it's like to compose for osu! tournaments:

> > For me, it would be between the composer(s) and the mappers to discuss about the base information of how the song should be, like whether snap divisions on this bpm are suitable to map in this difficulty before shaping my own ideas. I would also consider the song name, the background story and sometimes I would actually encourage the mappers to try feedback or mapping the demo to determine whether it is alright for the given difficulty.

Reviewer: ```suggestion
> For me, it would be between the composer(s) and the mappers to discuss about the base information of how the song should be, like whether snap divisions on this bpm are suitable to map in this difficulty before shaping my own ideas. I would also consider the song name, the background story, and sometimes I would actually encourage the mappers to try feedback or mapping the demo to determine whether it is alright for the given difficulty.
```

## A chaotic path of memories

Passion often comes with trials, and the most ambitious projects have by no means been easy achievements.

We'll leave you with what these creators dearly remember.

[![](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/Kasumi-sama.png)](https://osu.ppy.sh/users/6177263)

> \[...] I think the story behind [COEGTS](/wiki/Tournaments/GTS/COEGTS_2023)' Main Mappool Showcase and it getting finalized, like, at all, is one of my absolute favorite experiences as the main organizer of GTS.
> 
> First off, I believe everything related to creating this mappool showcase starts on July 10th 2023, being one day after our Qualifiers Mappool Showcase was released, and three weeks prior to the actual mappool showcase at [COE](/wiki/Community/cavoe's_osu!_event/2023), on July 31st.
> 
> At that point, we had most songs ready to go, except one was not given by the composer yet, and the tiebreaker had just gotten finalized that very day. 7 out of the 16 songs had an artwork for it made at all, and 2 of them had their video ready. Of course, lore was not even remotely created. As you can see, setting ourselves up for success here.
> 

> > The design channel then turned into an absolute warzone for the next 21 days, with designers, especially Transendium who has worked on 8 of these songs alone, managing to keep finalizing a few artworks here and there every day, even if some days were empty, due to them also working on other tournaments.

Reviewer: ```suggestion
> The design channel then turned into an absolute war zone for the next 21 days, with designers, especially Transendium who has worked on 8 of these songs alone, managing to keep finalizing a few artworks here and there every day, even if some days were empty, due to them also working on other tournaments.
```

> 
> We had gotten our last song for it to be mapped and make its video finalized on.. July 20th.
> 
> Fast forwarding again, we arrive at July 27th, where both of us admins, being myself and [TaikoMom](https://osu.ppy.sh/users/9086438), along with every single designer for GTS, our main voice actress for our female characters being [Ethereal_Winter](https://osu.ppy.sh/users/9780417), and some of our composers who were working on the lore part of things, being [blobdash](https://osu.ppy.sh/users/8337877) (for BGM), and NormalM (as a voice actor), literally stayed in VC for the next 4 days right up to the showcase day, where we kept rotating between everyone passing out at some point, but generally working on finalizing the design and videos for songs, writing the entire lore, recording all voicelines for lore between July 29th to July 30th, where I was forced to stay up until 9am to record voicelines LOL. It was honestly just a very fun experience overall.
> 

> > Except when we thought we finished the showcase entirely at 4am, when I personally had to go away to take my train for COE too, then we ended up realizing the overall mappool showcase video trolled us, with lore having had to be re-rendered twice, some gameplay being at 30FPS for no understandable reason, and TaikoMom then having to re-render all of that while I was gone, and needing to ensure we would actually have the correct video by uploading it on: MEGA / Google Drive / an USB key / Her Switch's SD card.

Reviewer: ```suggestion
> Except when we thought we finished the showcase entirely at 4am, when I personally had to go away to take my train for COE too, then we ended up realizing the overall mappool showcase video trolled us, with lore having had to be re-rendered twice, some gameplay being at 30FPS for no understandable reason, and TaikoMom then having to re-render all of that while I was gone, and needing to ensure we would actually have the correct video by uploading it on: MEGA / Google Drive / a USB key / Her Switch's SD card.
```

> 
> The most significant ones would be *Essence*, which I considered one of the significant moments of my life at that time. It was the biggest and the most emotional project we have ever done. I cannot stop going back and watch the reactions of the viewers at that time to the reveal of the tiebreaker song... It was both insane and emotional, and was the conclusion of the most productive session I've done at that time.
> 
> The other one which probably less or more related which is [osu!mania LN Tournament 3](https://www.youtube.com/watch?v=YcTvEi8ZPyA), I also did more than 10 custom maps for that tournament, including the custom song "[LNCryption](https://osu.ppy.sh/beatmapsets/1893946)", which was actually the space for the mappers to write and imagine the "hell" they faced somewhere else. It probably is worth to give them a read.
> 

> > I would also give an honorable mention to [rememberance; image of him before departure](https://osu.ppy.sh/beatmapsets/2028818) from Vietnamese National osu!mania Cup, which I composed in one day, during one of my depressive episodes. It was meant to be something that is lighthearted and healing me up. Quick compositions like this can also help me coping through hard times as well.

Reviewer: ```suggestion
> I would also give an honorable mention to [remembrance; image of him before departure](https://osu.ppy.sh/beatmapsets/2028818) from Vietnamese National osu!mania Cup, which I composed in one day, during one of my depressive episodes. It was meant to be something that is lighthearted and healing me up. Quick compositions like this can also help me coping through hard times as well.
```
